Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Definitions/HolographicFrameNativeData.cs

```diff
@@ -24,12 +24,12 @@ public struct HolographicFrameNativeData
         public uint MaxNumberOfCameras;
 
         /// <summary>
-        /// The current native root <see href="https://docs.microsoft.com/uwp/api/windows.perception.spatial.spatialcoordinatesystem">ISpatialCoordinateSystem</see>).
+        /// The current native root <see href="https://docs.microsoft.com/uwp/api/windows.perception.spatial.spatialcoordinatesystem">ISpatialCoordinateSystem</see>.
         /// </summary>
         public IntPtr ISpatialCoordinateSystemPtr;
         
         /// <summary>
-        /// The current native <see href="https://docs.microsoft.com/uwp/api/Windows.Graphics.Holographic.HolographicFrame">IHolographicFrame</see>).
+        /// The current native <see href="https://docs.microsoft.com/uwp/api/Windows.Graphics.Holographic.HolographicFrame">IHolographicFrame</see>.
```

Comment:
```suggestion
        /// The current native frame <see href="https://docs.microsoft.com/uwp/api/Windows.Graphics.Holographic.HolographicFrame">IHolographicFrame</see>.
```

Reply:
This change will resolve to `The current native frame IHolographicFrame` as opposed to just `The current native IHolographicFrame`, which I think is more clear.